copybookreader-plugins/src/main/java/co/cask/hydrator/plugin/batch/CopybookSource.java

```diff
@@ -86,24 +83,18 @@ public void initialize(BatchRuntimeContext context) throws Exception {
   public void prepareRun(BatchSourceContext context) throws IOException {
     Job job = JobUtils.createInstance();
     Configuration conf = job.getConfiguration();
```

Comment:
nit: let's move this after setting Job properties. Since you're passing Job into the InputFormat, it is a little more clear what is happening if you get the Configuration after you set everything on the Job.